features/print.yml

```
@@ -1,8 +1,6 @@
-name: Printing
-description: An alternative to `@media print` queries, the `window.print()` method, along with the `beforeprint` and `afterprint` events, allow you to change the page for printing, open the browser's print dialog, and restore the page after printing.
+name: window.print()
+description: The `window.print()` method opens the browser's print dialog, to give the user the choice to make a physical copy of the page.
```

I understand you wanted to not be circular, but print dialogs are often used to save PFDs too, so "make a physical copy" seems overly specific. Minimal fix is to shorten a lot:

```suggestion
description: The `window.print()` method opens the browser's print dialog.
```